docs/api/core.rst

```diff
@@ -200,27 +200,3 @@ Core events
 
 .. autoclass:: mopidy.core.CoreListener
     :members:
-
-Deprecated API features
```

Comment:
Switch to `Removed API features` and just listing the things that are gone? I.e. how do we reduce confusion from people not finding stuff we've removed?

I know this is in the changelog, is that enough?

Reply:
I believe it's better to remove it to avoid confusing visitors that searches for something, finds a result, and doesn't notice the header above what they're reading. I'd rather put in the effort to make the changelog or a migration guide really good. If searching the full docs site, I think it's best that the only hit for something removed is the changelog entry telling you that it was removed and what to use instead. If you need the full details of the removed APIS, docs for the previous release branches are also available.